db/migrate/20181128124306_assign_existing_exported_files_to_atos.rb

```diff
@@ -0,0 +1,19 @@
+class AssignExistingExportedFilesToAtos < ActiveRecord::Migration[5.2]
```

Comment:
records in the exported_files table need to have the external_system_id set (enforced in later migration). This migration fixes existing data